site-scrapers/MAImmunizations.js

```diff
@@ -43,11 +43,13 @@ async function ScrapeWebsiteData(browser) {
 			const date = title.substring(onIndex + 4);
 
 			//address like [STREET], [CITY] MA, [ZIP]
+			//address like [STREET], [CITY] Ma, [ZIP]
+			//address like [STREET], [CITY] Massachusetts, [ZIP]
 			const firstCommaIndex = address.indexOf(", ");
 			const street = address.substring(0, firstCommaIndex);
-			let stateIndex = address.indexOf(" MA");
+			let stateIndex = address.toUpperCase().indexOf(" MA,");
```

Comment:
We should remove this comma and change it to lastIndexOf (so we don't break on Malden). I had a comma in before (like a few weeks ago) and there was at least one location where there was no comma after MA 🤦‍♀️